src/Foundation/NSMetadataItem.cs

```diff
@@ -24,6 +24,16 @@ bool GetBool (NSString key)
 			return n == null ? 0 : n.DoubleValue;
 		}
 
+		nint GetNInt (NSString key)
+		{
+			var n = Runtime.GetNSObject<NSNumber> (GetHandle (key));
+#if XAMCORE_2_0
+			return n?.NIntValue ?? 0;
```

Comment:
I basically just copied the GetDouble method above for this one (presumably they should act the same).  I'm not entirely sure how to determine whether they would default to 0 or null.